src/app/apiFetch/User.ts

```diff
@@ -180,3 +180,20 @@ export const checkUsernameExists = (username: string) => {
       console.error(error);
     });
 };
+
+export const userGetQuestStatus = () => {
+  console.log('Inside apiFetch');
```

Comment:
Why is console.log here? Are pull requests being reviewed @ram2091999 @bestinbthomas @ElzaCS ?